browser/ui/views/frame/brave_browser_view.h

```diff
@@ -65,6 +65,8 @@ class BraveBrowserView : public BrowserView {
       const TabStripSelectionChange& selection) override;
 
   void StopTabCycling();
+  PrefChangeRegistrar pref_change_registrar_;
```

Comment:
We declare method and variables separately.
Methods are declared and then variables are.
This variable decl should go around below `tab_cycling_event_handler_`.